addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -6,17 +6,42 @@
     display: flex;
     flex-direction: row;
     justify-content: space-between;
-}
 
-.o_ChatterTopbar_buttonCount {
-    padding-left: 0.25rem !important;
+    .o_ChatterTopbar_buttonAttachmentsCountLoader {
+        margin-left: 2px;
+    }
+
+    .o_ChatterTopbar_buttonCount {
+        padding-left: 0.25rem !important;
+    }
+
+    .o_ChatterTopbar_buttonSendMessage {
+        padding: 0 16px;
+        border-left: none !important;
```

Comment:
why do you need `!important`? Ideally you should not use this, but I assume this is to override a bootstrap style. See which style is problematic and solve with more specific style-rule without `!important`

Reply:
Just something I forgot to remove (I was struggling with a problem of ignore scss rules)